topic_resources/javascript/exercises-101.md

```diff
@@ -16,6 +16,6 @@ Write a function called reverseArray() that takes an array, `ar` as an argument
 
 Write a function called reverseArrayInPlace() that takes an array `ar` as an argument and returns the same array modified to have its elements in reverse order.
```

Comment:
This seems a little unclear as to whether the passed array should be modified or not (or it doesn't matter).